json/targeting.yaml

```diff
@@ -97,6 +97,22 @@ type: object
         - type: array
           items:
             type: string
+  ifRule:
```

Comment:
With this change, "if" can no longer share the its anatomy with other ops (`binaryOrTernaryOp`), since it's the only one that takes 1-3 args, so it's implemented as it's own rule now.